features/FxA-45-device-registration-api/README.md

```diff
@@ -0,0 +1,360 @@
+# Device registration API
+
+https://mozilla.aha.io/features/FXA-45
+
+As a Firefox developer, I want
+to be able to associate device details with an FxA login session,
+and receive notifications when those details are changed on the server,
+so that I can start building richer device-driven features
+for the Firefox Connected Experience.
+
+* [Stories](#stories)
+* [Data](#data)
+* [API](#api)
+  * [POST /v1/account/login](#post-v1accountlogin)
+  * [POST /v1/account/create](#post-v1accountcreate)
+  * [POST /v1/account/device](#post-v1accountdevice)
+  * [POST /v1/account/device/destroy](#post-v1accountdevicedestroy)
+  * [GET /v1/account/devices](#get-v1accountdevices)
+* [Details](#details)
+  * [What is a "recovering device"?](#what-is-a-recovering-device)
+  * [How are shared devices handled?](#how-are-shared-devices-handled)
+  * [Are shared devices bound to the account or the device?](#are-shared-devices-bound-to-the-account-or-the-device)
+  * [How are unrecognised devices handled?](#how-are-unrecognised-devices-handled)
+  * [How big should the device id be?](#how-big-should-the-device-id-be)
+
+## Stories
+
+Is it reasonable to break these down
```

Comment:
Breaking the stories down one per endpoint seems like a good approach to me but I wasn't sure whether you guys would agree, which is why I asked the question.